src/ocp_postprocess/hostname_rename/filesystem_rename.rs

```diff
@@ -228,3 +229,35 @@ pub(crate) async fn fix_filesystem_kapi_startup_monitor_configmap_pod_yaml(
 
     Ok(())
 }
+
+pub(crate) async fn fix_filesystem_etcd_all_certs(original_hostname: &str, hostname: &str, dir: &Path) -> Result<()> {
+    join_all(
+        file_utils::globvec(dir, "**/etcd-certs/secrets/etcd-all-certs/etcd-*")?
+            .into_iter()
+            .map(|file_path| {
+                let original_path = file_path.clone();
+                let new_path = file_path
+                    .clone()
+                    .into_os_string()
+                    .into_string()
+                    .unwrap()
+                    .replace(original_hostname, hostname);
+                tokio::spawn(async move {
+                    async move {
+                        fs::rename(original_path, new_path)?;
```

Comment:
we should have a function similar to commit_file so it can be avoided during a dry run

Reply:
Of course, I completely neglected the dry run option. I'll try to update it accordingly.

Reply:
I didn't add a function like `commit_rename` because we only need it here, but I can do that if you think it makes sense for the long term. WDYT?

Reply:
I prefer a separate function just because I don't want to concern rename code with the concept of dry run

Reply:
But I'll just do it separately later as part of a refactor of all this code, too much repetition. I'm afraid to touch it because we have no IBI/IBU tests

Reply:
Got it, this makes sense, I'm working on adding the recert hostname change 0-rollout CI job (not IBI/IBU at this point, just recert, but I think until we have an IBI/IBU e2e job, we can already increase our confidence starting with this recert specific job).